packages/replay/src/util/isIngestHost.ts

```diff
@@ -1,18 +0,0 @@
-import { getCurrentHub } from '@sentry/core';
-
-import { isInternal } from './isInternal';
```

Comment:
Is this still used elsewhere? Can the file be removed too?

Reply:
Oops, you're right, completely forgot this!